packages/client-generator-ts/src/generator.ts

```diff
@@ -13,23 +14,35 @@ import { resolvePrismaClient } from './resolvePrismaClient'
 const debug = Debug('prisma:client:generator')
 
 type PrismaClientTsGeneratorOptions = {
-  shouldResolvePrismaClient?: boolean
   runtimePath?: string
 }
 
+const missingOutputErrorMessage = `An output path is required for the \`prisma-client-ts\` generator. Please provide an output path in your schema file:
+
+${dim(`generator client {
+  provider = "prisma-client-ts"`)}
+${green('  output   = "../src/generated"')}
```

Comment:
It may be worth mentioning what directory the output path is relative to, since you used `..` in the example directory path. I guess this is relative to the schema file's directory.

Reply:
great point

Reply:
done
<img width="754" alt="image" src="https://github.com/user-attachments/assets/63632f23-8ef7-4be8-a4f7-50b5286e29a0" />